src/components/CreateRules/UploadRules.vue

```diff
@@ -201,9 +200,10 @@ export default class UploadRules extends Mixins(CommonMixin, DocumentMixin) {
   private INPUT_FILE_LABEL = 'Rules of Association'
   private hasValidUploadFile: boolean = false
   private hasRulesConfirmed: boolean = false
-  private rulesConfirmed = false
+  private rulesConfirmed: boolean = false
```

Comment:
You don't to define it as boolean if you assign it to a boolean. You can remove the other boolean types above.

Reply:
removed boolean types